pkg/gadgets/advise/seccomp/tracer/tracer.go

```diff
@@ -78,24 +80,41 @@ func NewTracer() (*Tracer, error) {
 	return t, nil
 }
 
-func (t *Tracer) Peek(mntns uint64) []byte {
+func syscallArrToNameList(v []byte) []string {
+	names := []string{}
+	for i, val := range v {
+		if val == 0 {
+			continue
+		}
+		call1 := libseccomp.ScmpSyscall(i)
+		name, err := call1.GetName()
+		if err != nil {
+			name = fmt.Sprintf("syscall%d", i)
+		}
+		names = append(names, name)
+	}
+	sort.Strings(names)
+	return names
+}
+
+func (t *Tracer) Peek(mntns uint64) []string {
 	b, err := t.seccompMap.LookupBytes(mntns)
 	if err != nil {
 		log.Errorf("Error while looking up the seccomp map: %s", err)
-		return make([]byte, C.SYSCALLS_COUNT)
+		return make([]string, 0)
```

Comment:
I think this should return the error here so the caller can handle it. (I know it's not related to this PR but I think we can take the opportunity to fix it). Same in other places.